src/main/java/com/google/opensesame/servlets/PersonObject.java

```diff
@@ -22,17 +22,19 @@
    *
    * @param gitHubID
    * @param interestTags
+   * @param email
    * @throws IOException
    */
-  public PersonObject(String gitHubID, ArrayList<String> interestTags) throws IOException {
+  public PersonObject(String gitHubID, ArrayList<String> interestTags, String email)
+      throws IOException {
```

Comment:
Can we also add the userId to the person object? I have a feeling a lot of this code is going to be torn down on the transition to Objectify so feel free to add it then (I think you need to add the ID for Objectify, see [here](https://github.com/objectify/objectify/wiki/Entities#the-basics)):

> Entities must have one field annotated with @Id. The actual name of the field is irrelevant and can be renamed at any time, even after data is persisted. This value (along with the kind 'Car') becomes part of the Key which identifies an entity.

Reply:
On this note, are you planning on deprecating PersonObject eventually? It seems that PersonObject and PersonEntity are nearly identical so we might only need one.